drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
```

Comment:
I see that you have exactly retained the functionality as the old driver :-). But in this case I think it is wrong to not add the route if the bridge existed. If the bridge existed and if it didn't have the route is it ok not to have the route? I think not.